exporter/signalfxexporter/exporter.go

```diff
@@ -175,47 +194,62 @@ func newGzipPool() sync.Pool {
 	}}
 }
 
-func newEventExporter(config *Config, logger *zap.Logger) (*signalfxExporter, error) {
+func newEventExporter(config *Config, createSettings exporter.CreateSettings) (*signalfxExporter, error) {
 	if config == nil {
 		return nil, errors.New("nil config")
 	}
 
-	options, err := config.getOptionsFromConfig()
+	_, err := config.getOptionsFromConfig()
```

Comment:
Is this required anymore?

I imagine if there is an issue processing the config, it should be caught in `config.Validate()`

Reply:
it's a bit contrived, but I wanted to make this without changing the tests too much. Agree we can refactor more, but that felt like a bit more work than what I was ready to deliver.